python_base/Dockerfile

```diff
@@ -42,8 +42,11 @@ RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.n
         python-virtualenv \
         wget && \
     yum clean all
-RUN npm install -g \
-        node-sass \
+RUN npm cache clean -f && \
+    npm install -g n && \
+    n 4.4.6 && \
```

Comment:
Using the version 3.8.0 of node-sass  seems to be working for me locally without needing to downgrade node itself.